app/docs/0.10.x/admin-api.md

```diff
@@ -1339,7 +1339,8 @@ service. Every upstream can have many targets, and the targets can be
 dynamically added and removed. So changes are effectuated on the fly.
 
 Because the upstream maintains a history of target changes, the targets cannot
-be deleted or modified. To disable a target, post a new one with `weight=0`.
+be deleted or modified. To disable a target, post a new one with `weight=0`, or
+use the `DELETE` sugar method to accomplish the same.
```

Comment:
Even though this is sugar, do we use the word "sugar" here? Not sure what is normal.